tox.ini

```
@@ -7,18 +7,9 @@ envlist =
 requires =
     setuptools >= 30.3.0
     pip >= 19.3.1
-    tox-pypi-filter >= 0.12
-    tox < 4
-isolated_build = true
 
 [testenv]
 
-# The following option combined with the use of the tox-pypi-filter above allows
-# project-wide pinning of dependencies, e.g. if new versions of pytest do not
-# work correctly with pytest-astropy plugins. Most of the time the pinnings file
-# should be empty.
-pypi_filter = https://raw.githubusercontent.com/astropy/ci-helpers/main/pip_pinnings.txt
-
 # Pass through the following environment variables which are needed for the CI
 passenv = HOME WINDIR LC_ALL LC_CTYPE CC CI IS_CRON ARCH_ON_CI
```

```suggestion
passenv = HOME,WINDIR,LC_ALL,LC_CTYPE,CC,CI,IS_CRON,ARCH_ON_CI
```
changed syntax in tox 4